blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -98,21 +97,18 @@ def _on_service_property_changed(self, _service, key, _value, _path):
         if key == "Connected":
             self.generate()
 
-    def on_connect(self, _item, service):
+    def on_connect(self, _item: Gtk.MenuItem, service: Service) -> None:
         device = service.device
 
-        def success(obj, result, _user_data):
+        def success(_obj: AppletService, _result: None, _user_data: None) -> None:
             logging.info("success")
             prog.message(_("Success!"))
 
-            if isinstance(service, SerialPort) and SERIAL_PORT_SVCLASS_ID == service.short_uuid:
-                MessageArea.show_message(_("Serial port connected to %s") % result, None, "dialog-information")
```

Comment:
Does the user get feedback that the serial port got connected?

Reply:
`SerialManager.on_rfcomm_connected` triggers a notification for that (That's actually a little problem if `PPPSupport` handles a dialup connection and `SerialManager` is not loaded, however, a message saying that a serial port got connected to None won't be helpful either. :laughing:). Also see first commit now.